src/app/shared/services/user-session.service.ts

```diff
@@ -65,6 +77,12 @@ export class UserSessionService implements OnDestroy {
     return !session || session.user.tempUser;
   }
 
+  updateCurrentUser(changes: UpdateUserBody): Observable<void> {
+    const update$ = this.currentUserService.update(changes).pipe(shareReplay());
```

Comment:
I think you have to mention the size of the buffer
```suggestion
    const update$ = this.currentUserService.update(changes).pipe(shareReplay(1));
```

Reply:
The buffer size is optional, but one could argue that providing the buffer size is more explicit for readability.

From a code point of view, I'm not sure it matters much. There's one value only in this observable, which will be repeated. If someday there are more values, they will all be repeated with current share replay config. From this point of view it looks fine to me.

I'll change it anyway to avoid starting a discussion that might be more entertaining than useful.
Fixed